pkg/pod/pod.go

```diff
@@ -18,6 +18,7 @@ package pod
 
 import (
 	"fmt"
+	"github.com/tektoncd/pipeline/pkg/version"
```

Comment:
this needs to be with the other `github.com/…` imports (`goimports` should do that automatically)